### PR TITLE
added 2023-24 billing slab

### PR DESCRIPTION
**Pull request by Saloni-eGov**

### CODE REVIEW

File: data/pb/BillingService/TaxPeriod.json [hunk 1]
```diff
@@ -169,6 +169,94 @@
...
+      "service": "EXPENSE.OTHERS",
+      "code": "EXPENSEOTHERS",
+      "financialYear": "2023-24"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "TWOYEARS",
+      "service": "WS",
+      "code": "WC2021",
```
Comment (egov-arindam) [inline data/pb/BillingService/TaxPeriod.json:210]: @Saloni-eGov Please change the code of "code": "WC2021" to "code": "WC2023"

File: data/pb/BillingService/TaxPeriod.json [hunk 2]
```diff
@@ -169,6 +169,94 @@
...
+      "toDate": 1711866599000,
+      "periodCycle": "ANNUAL",
+      "service": "EXPENSE.OTHERS",
+      "code": "EXPENSEOTHERS",
+      "financialYear": "2023-24"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "TWOYEARS",
```
Comment (egov-arindam) [inline data/pb/BillingService/TaxPeriod.json:208]: @rahuldevgarg-egov @Saloni-eGov WHat is the use of periodCycle in MDMS. Please check this.

Comment (rahuldevgarg-egov) [reply data/pb/BillingService/TaxPeriod.json:208]:
```
if (taxPeriodCriteria.getPeriodCycle() != null) {
			if (filterExpression.length() != 0)
				filterExpression.append(" && ");
			filterExpression
					.append(TAXPERIOD_PERIODCYCLE_FILTER.replace("VAL", taxPeriodCriteria.getPeriodCycle().toString()));
		}
```
it is used in MDMS service for getting List of TaxPeriod

File: data/pb/BillingService/TaxPeriod.json (hunk 1 above)
Comment (rahuldevgarg-egov) [reply data/pb/BillingService/TaxPeriod.json]: marking as resolved. Changes are done by @Saloni-eGov

**Pull request merged**